app/controllers/admin/users_controller.rb

```diff
@@ -34,6 +36,7 @@ def create
       if @user.save
         format.html { redirect_to admin_users_path, notice: 'User was successfully created.' }
         format.json { render :show, status: :created, location: @user }
+        after_create :send_admin_mail
```

Comment:
Does this work? Typically in controllers you just need to call the method:

`send_admin_mail`